clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2014,23 +1953,48 @@ class SyclKernelDeclCreator : public SyclKernelFieldHandler {
         SYCLIntelBufferLocationAttr::CreateImplicit(Ctx, LocationID));
   }
 
+  // Handle accessor's parameters for both, accessor as a field to a base class,
```

Comment:
I think saying `accessor's parameters` might be confusing, since it is actually `__init` function's parameters. How about just saying - "Additional processing required for accessor type"